dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/EventValidator.java

```diff
@@ -62,15 +63,17 @@ public EventValidator(
                     new UpdatableFieldsValidator(),
                     new DataRelationsValidator(),
                     new CategoryOptionComboValidator(),
-                    securityOwnershipValidator,
+                    seq(
```

Comment:
I don't think you need to group them in a `seq` as they are already in one so will run sequentially and fail on the first one 🤔